src/storm/modelchecker/helper/infinitehorizon/SparseDeterministicInfiniteHorizonHelper.cpp

```diff
@@ -529,8 +530,8 @@ std::vector<ValueType> SparseDeterministicInfiniteHorizonHelper<ValueType>::comp
 }
 
 template<typename ValueType>
-std::vector<ValueType> computeUpperBoundsForExpectedVisitingTimes(storm::storage::SparseMatrix<ValueType> const& nonBsccMatrix,
-                                                                  std::vector<ValueType> const& toBsccProbabilities) {
+std::vector<ValueType> computeUpperBoundsForExpectedVisitingTimes([[maybe_unused]] storm::storage::SparseMatrix<ValueType> const& nonBsccMatrix,
+                                                                  [[maybe_unused]] std::vector<ValueType> const& toBsccProbabilities) {
```

Comment:
Because I mixed up this function with the one below it, where the `[[maybe_unused]]` was actually supposed to go. Working on fixing it, thanks for catching that error!